cmd/root.go

```diff
@@ -68,19 +56,9 @@ EXAMPLES
 	viper.AutomaticEnv()       // read in environment variables for FUNC_<flag>
 	viper.SetEnvPrefix("func") // ensure that all have the prefix
 
-	// Flags
-	// persistent flags are available to all subcommands implicitly
-	// Note they are bound immediately here as opposed to other subcommands
-	// because this root command is not actually executed during tests, and
-	// therefore PreRunE and other event-based listeners are not invoked.
-	cmd.PersistentFlags().BoolP("verbose", "v", false, "Print verbose logs ($FUNC_VERBOSE)")
-	if err := viper.BindPFlag("verbose", cmd.PersistentFlags().Lookup("verbose")); err != nil {
-		fmt.Fprintf(os.Stderr, "error binding flag: %v\n", err)
-	}
-
 	// Version
-	cmd.Version = cfg.Version.String()
-	cmd.SetVersionTemplate(`{{printf "%s\n" .Version}}`)
+	// cmd.Version = cfg.Version.String()
+	// cmd.SetVersionTemplate(`{{printf "%s\n" .Version}}`)
```

Comment:
Are these meant to be commented out?

Reply:
Ah yes, thank you for noticing. 
I was hemming-and-hawing about whether or not to leave it in there.